src/arch/armv8/inc/arch/gicv3.h

```diff
@@ -107,6 +107,8 @@ static inline void gich_write_lr(size_t i, uint64_t val)
         case 15:
             sysreg_ich_lr15_el2_write(val);
             break;
+        default:
+            break;
```

Comment:
Shouldn't this return an error / hang? or at least a warning

Reply:
We've debated this. But is is just maintaining the same behavior as before. Maybe we could "fix" it in another PR.